### ISSUE TITLE
Switching to bold changes line spacing (in Pages, Albert Sans v1.3)

### ISSUE THREAD

**Issue by HolgerGerhardt** [usted/Albert-Sans#4]
Selecting bold in the Pages app (macOS) changes the line spacing as long as I do not set the line spacing to a fixed value in points:

<img width="433" alt="AlbertWordPages" src="https://github.com/user-attachments/assets/19f97d1c-e368-452e-8b86-121eaece4c5e">

(MS Word for comparison on the left, Pages on the right. Libre Office Writer also does not display this issue.)

**Comment by kenmcd** [comment]
Looks like an error in Pages.
The vertical metrics are identical in the Regular and Bold fonts.

**Comment by HolgerGerhardt** [comment]
> Looks like an error in Pages.

I have tested dozens of other fonts. For none of them does this happen—except one: FiraGO (https://github.com/bBoxType/FiraGO).

**Comment by usted** [comment]
This is a strange issue that is due to mac os system. It can be fixed by including The STAT table in the font files. Unfortunately GF ignores that tag and doesn't include it. So to make a Long story short: i have fixed The issue by lowering The height of The accents.
So it's fixed in The new release. 
Thank you both.

**Comment by kenmcd** [comment]
> This is a strange issue that is due to mac os system. It can be fixed by including The STAT table in the font files.

Interesting. 
Do you have a background link for this recommendation?
I would like see more info.
Note: GF has been including STAT tables in their static fonts now.
Wonder if this is part of the reason why.

**Comment by usted** [comment]
Yes, you can see it here: https://forum.glyphsapp.com/t/changing-position-in-bold-vs-regular/30923/11
When I've added The "Export STAT table"-tag it's ignored by GF build and the STAT table are not included so I only see lowering the accents to work around this.

**Comment by HolgerGerhardt** [comment]
> So it's fixed in The new release.

Thank you, @usted! I can confirm that the issue is fixed. (I checked both the OTF and TTF version.)

(Funnily, the Pages default font, Helvetica Neue, suffers from the same issue … With Helvetica Neue, even selecting italics changes the line height.)
